.github/CODEOWNERS

```
@@ -36,6 +36,8 @@
 /addons/*/models/ir_qweb.py @odoo/rd-website
 /addons/*/models/ir_qweb_fields.py @odoo/rd-website
 /addons/website*/ @odoo/rd-website
+/addons/website_event*/ @rd-notif-muted
+/addons/website_slides*/ @rd-notif-muted
```

@robodoo r-

Slight adaption though, teams should be prefixed by the organisation, otherwise it is targeting the github user @rd-notif-muted
```suggestion
/addons/website_event*/ @odoo/rd-notif-muted
/addons/website_slides*/ @odoo/rd-notif-muted
```